public/assets/code/colouredtriangles/sol.py

```diff
@@ -0,0 +1,12 @@
+inp = input()
+curr = list(inp)
+
+while (len(curr) > 1):
+    new = []
+    for i in range(0, len(curr)-1):
+        c1, c2 = curr[i], curr[i+1]
+        if c1 == c2: new.append(c1)
+        else: new.append(list({c1, c2}^{"R", "G", "B"})[0])
```

Comment:
Is there a simpler way of doing this? As a non-python user, I don't know what the ^ operator does between sets. I can take a guess, but I feel like the average person reading this editorial (an extremely simple problem) won't be very good at python and would have a hard time understanding this line of code